src/_includes/cwl/hello-world-expressionlib.cwl

```diff
@@ -0,0 +1,25 @@
+cwlVersion: v1.2
+class: CommandLineTool
+requirements:
+  - class: InlineJavascriptRequirement
+    expressionLib:
+      - { $include: functions.js }
+      - |
+        /**
+         * An example function that uses a function from the included
+         * functions.js file to create a Hello World message.
+         * @param {Object} message - CWL document input message
+         */
+        var createHelloWorldMessage = function (message) {
+          return capitalizeWords(message);
+        };
```

Comment:
Interesting to show a mixture of includes and inline; might be better to show each case separately?

Reply:
Can't take credit for the example, I modeled it after this example https://github.com/common-workflow-language/common-workflow-language/issues/788#issuecomment-435007087 :slightly_smiling_face: 

I was trying to make this example simpler. If we break it down in two examples, would it be for the same code?

I liked this example as it shows that whatever is loaded in an entry of `expressionLib` gets added to the global context and will be available for other entries of `expressionLib`, but also to other parts of the code where Expressions are supported.

If we break it down into two examples, I think it would be nice to show

- just `$include`
- just inline JavaScript
- both combined

I think that section would get a bit longer, and the user would get the same idea in both cases… I think… WDYT?